dbbuild/build-summary.txt

```diff
@@ -70,66 +70,378 @@ swapping out rs28970291 for rs4076758
+swapping out rs281863375 for rs9274225
+swapping out rs281863371 for rs41270933
+swapping out rs281863364 for rs17613629
+swapping out rs281863363 for rs17613636
+swapping out rs281863357 for rs9274227
+swapping out rs115980227 for rs80319605
+swapping out rs115980227 for rs80319605
+swapping out rs536547537 for rs80087112
+swapping out rs536547537 for rs80087112
+swapping out rs281863504 for rs71542446
+swapping out rs281863503 for rs41270931
+swapping out rs281863502 for rs28724261
+swapping out rs281863499 for rs71542447
+swapping out rs281863486 for rs2854267
+swapping out rs281863485 for rs9274216
+swapping out rs281863467 for rs201930518
+swapping out rs281863466 for rs17613599
+swapping out rs281863464 for rs71542448
+swapping out rs281863463 for rs9274217
+swapping out rs281863439 for rs58770498
+swapping out rs281863431 for rs17613606
+swapping out rs281863430 for rs9274218
+swapping out rs281863427 for rs115495316
+swapping out rs281863426 for rs9274219
+swapping out rs281863504 for rs71542446
+swapping out rs281863503 for rs41270931
+swapping out rs281863502 for rs28724261
+swapping out rs281863499 for rs71542447
+swapping out rs281863486 for rs2854267
+swapping out rs281863485 for rs9274216
+swapping out rs281863467 for rs201930518
+swapping out rs281863466 for rs17613599
+swapping out rs281863464 for rs71542448
+swapping out rs281863463 for rs9274217
+swapping out rs281863439 for rs58770498
+swapping out rs281863431 for rs17613606
+swapping out rs281863430 for rs9274218
+swapping out rs281863427 for rs115495316
+swapping out rs281863426 for rs9274219
+swapping out rs576530137 for rs281864341
+swapping out rs530460066 for rs281864275
+swapping out rs576530137 for rs281864341
+swapping out rs530460066 for rs281864275
+swapping out rs573533995 for rs12222263
+swapping out rs573533995 for rs12222263
+swapping out rs536547537 for rs80087112
+swapping out rs536547537 for rs80087112
 Merging 'mh01WL-001' --> 'mh01KK-212'
-Merging 'mh01WL-012' --> 'mh01KK-211'
+Merging 'mh01WL-062' --> 'mh01KK-212'
+Merging 'mh01WL-048' --> 'mh01KK-212'
+Merging 'mh01WL-082' --> 'mh01WL-020'
+Merging 'mh01WL-049' --> 'mh01WL-003'
+Merging 'mh01WL-068' --> 'mh01WL-033'
+Merging 'mh01WL-061' --> 'mh01WL-024'
+Merging 'mh01WL-050' --> 'mh01WL-024'
+Merging 'mh01WL-064' --> 'mh01WL-027'
+Merging 'mh01WL-060' --> 'mh01WL-047'
+Merging 'mh01WL-065' --> 'mh01WL-002'
+Merging 'mh01WL-055' --> 'mh01WL-002'
+Merging 'mh01WL-079' --> 'mh01WL-040'
+Merging 'mh01WL-075' --> 'mh01WL-010'
 Merging 'mh01NK-001' --> 'mh01NH-04'
 Merging 'mh01WL-057' --> 'mh01ZHA-012'
+Merging 'mh01WL-054' --> 'mh01WL-041'
+Merging 'mh01WL-085' --> 'mh01WL-019'
+Merging 'mh01WL-012' --> 'mh01KK-211'
+Merging 'mh01WL-072' --> 'mh01KK-211'
+Merging 'mh01WL-067' --> 'mh01WL-008'
+Merging 'mh01WL-056' --> 'mh01WL-008'
+Merging 'mh01WL-069' --> 'mh01WL-005'
 Merging 'mh01WL-096' --> 'mh01FHL-009'
+Merging 'mh01WL-066' --> 'mh01FHL-009'
+Merging 'mh01WL-063' --> 'mh01WL-088'
```

Comment:
Lots of merges within the study (between different groups) and with other studies.